app/models/katello/concerns/smart_proxy_extensions.rb

```diff
@@ -129,6 +129,19 @@ def pulp3_configuration(config_class)
         end
       end
 
+      def pulp3_cert_configuration(config_class)
+        cert_config = config_class.new do |config|
+          uri = pulp3_uri!
+          config.host = uri.host
+          config.scheme = uri.scheme
+          config.ssl_client_cert = ::Cert::Certs.ssl_client_cert
+          config.ssl_client_key = ::Cert::Certs.ssl_client_key
+          config.debugging = true
+          config.logger = ::Foreman::Logging.logger('katello/pulp_rest')
+        end
+        cert_config
+      end
```

Comment:
what is this used for exactly? (how is it different than the pulp3_configuration method?

Reply:
I was seeing some issues with configuring the cert_guard. This was in trying to work around that..Will test as this might not be needed any more..

Reply:
This isn't needed..Tested with the default pulp_config method.